### DISCUSSION TITLE
Demo on esp32-4827S043

### DISCUSSION THREAD

**Discussion by pieterjm** [rzeldent/esp32-smartdisplay#131] [Ideas]
Hi, and thanks for the great library!

I had to add the following line to get the demo running on the esp32-4827S043C board.

smartdisplay_lcd_set_backlight(1);

 At first, the screen remains dark all the time. Also tested several branches (main, develop and release/2.0.5) all with the same results. 

Serial monitor shows that Touches are registered. I'm seeing a lot of gt911_lvgl_touch_cb() events when I touch the screen. 

When inspecting the serial monitor log, i saw that the lcd backlight duty cycle was set to 0.5. Changing that into 1 solved my issue.

**Comment by AlecDavisNZ** [comment]
I found exactly the same with the esp32-4848S040CIY3, touches are registered, but screen was blank.

Adding smartdisplay_lcd_set_backlight(1); after smartdisplay_init(); turned the display on.
But it doesn't feel the right way.

**Comment by rzeldent** [comment]
Was caused by the PWM frequency that was too high. Should be solved in current versions...

Enjoy!
